ballerina/Ballerina.toml

```diff
@@ -0,0 +1,77 @@
+[package]
+org = "ballerinax"
+name = "wso2.apim.catalog"
+version = "0.1.0"
+distribution = "2201.9.0-SNAPSHOT"
+
+[[platform.java17.dependency]]
+groupId = "io.ballerina"
+artifactId = "wso2.apim.catalog-native"
+version = "0.1.0"
+path = "../native/build/libs/wso2.apim.catalog-native-0.1.0.jar"
+
+[[platform.java17.dependency]]
+groupId = "com.fasterxml.jackson.core"
+artifactId = "jackson-core"
+version = "2.15.2"
+path = "lib/jackson-core-2.15.2.jar"
+
+[[platform.java17.dependency]]
+groupId = "com.fasterxml.jackson.core"
+artifactId = "jackson-databind"
+version = "2.15.2"
+path = "lib/jackson-databind-2.15.2.jar"
+
+[[platform.java17.dependency]]
+groupId = "com.fasterxml.jackson.core"
+artifactId = "jackson-annotations"
+version = "2.15.2"
+path = "lib/jackson-annotations-2.15.2.jar"
+
+[[platform.java17.dependency]]
+groupId = "com.fasterxml.jackson.datatype"
+artifactId = "jackson-datatype-jsr310"
+version = "2.15.2"
+path = "lib/jackson-datatype-jsr310-2.15.2.jar"
+
+[[platform.java17.dependency]]
+groupId = "com.fasterxml.jackson.dataformat"
+artifactId = "jackson-dataformat-yaml"
+version = "2.15.2"
+path = "lib/jackson-dataformat-yaml-2.15.2.jar"
+
+[[platform.java17.dependency]]
+groupId = "io.swagger.core.v3"
+artifactId = "swagger-core-jakarta"
+version = "2.2.10"
+path = "lib/swagger-core-jakarta-2.2.10.jar"
+
+[[platform.java17.dependency]]
+groupId = "io.swagger.core.v3"
+artifactId = "swagger-models"
+version = "2.2.10"
+path = "lib/swagger-models-2.2.10.jar"
+
+[[platform.java17.dependency]]
+groupId = "io.swagger.parser.v3"
+artifactId = "swagger-parser-v3"
+version = "2.2.10"
+path = "lib/swagger-parser-v3-2.1.16.jar"
+
+[[platform.java17.dependency]]
```

Comment:
Please check weather we need both of these swagger parser dependencies.

Reply:
Yes, we need